src/jit/instrsxarch.h

```diff
@@ -217,6 +217,12 @@ INST3( shufpd,      "shufpd"      , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE, PCK
 
 INST3( punpckhdq,   "punpckhdq"   , 0, IUM_WR, 0, 0, BAD_CODE,     BAD_CODE, PCKDBL(0x6A))
 
+INST3( prefetchnta, "prefetchnta" , 0, IUM_RD, 0, 0, 0x000F0018,   BAD_CODE, BAD_CODE)
+INST3( prefetcht0,  "prefetcht0"  , 0, IUM_RD, 0, 0, 0x000F0818,   BAD_CODE, BAD_CODE)
+INST3( prefetcht1,  "prefetcht1"  , 0, IUM_RD, 0, 0, 0x000F1018,   BAD_CODE, BAD_CODE)
+INST3( prefetcht2,  "prefetcht2"  , 0, IUM_RD, 0, 0, 0x000F1818,   BAD_CODE, BAD_CODE)
+INST3( sfence,      "sfence"      , 0, IUM_RD, 0, 0, 0x000FF8AE,   BAD_CODE, BAD_CODE)
```

Comment:
These instructions are not SSE/VEX-encoding, we should put them out of this area (like `crc32`).

Reply:
Yes, but they are SSE/SSE2 instructions and so I put them into the FIRST_SSE2_INSTRUCTION, LAST_SSE2_INSTRUCTION range. (I'm also not sure they aren't SSE-encoding, they all have the `0F` prefix that SSE instructions use and are part of that ISA, I just didn't use `the SSEFLT helper since manually encoding the third byte is more useful).

Maybe @CarolEidt could comment as to whether the range is for all SSE/SSE2 instructions, or just those that should get the VEX prefix...

Reply:
I just worried about that we need to specially treat these instructions everywhere if they are in this range (but with special encoding rules). If these instructions do not mess other ones (just disable VEX encoding), that would be good.

Reply:
It looks like the only place is the `TakesVexPrefix` check. If there are other issues, I would imagine we could clean them up by making this more table driven (which has at least one tracking issue)

Reply:
> Maybe @CarolEidt could comment as to whether the range is for all SSE/SSE2 instructions, or just those that should get the VEX prefix...

IIRC this category existed prior to the addition of VEX encoding support, but I'm not certain.